podman/tinyproxy/dev.sh

```diff
@@ -0,0 +1,3 @@
+docker build . -t tp
+docker run -p 23128:8888 --name=tp -v $(pwd)/conf/:/usr/local/etc/tinyproxy/:z tp
+docker rm -fv tp
```

Comment:
docker run --rm -p 23128:8888 --name=tp -v $(pwd)/conf/:/usr/local/etc/tinyproxy/:z tp

^ Why not just this?

Reply:
<facepalm>  ...thanks.  :)